### PR TITLE
Custom Routines - Prediction Sample

### PR DESCRIPTION
**Pull request by happyhuman**

### CODE REVIEW

File: prediction/sklearn/structured/custom_routines/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Online Prediction with scikit-learn and Custom Routines
+
+## Overview
+
+In this tutorial, we go over the steps to deploy a model trained with scikit-learn to AI-Platform.
+We will be using the model that was trained using custom routines. We will deploy the model and pass
```
Comment (nnegrey) [inline prediction/sklearn/structured/custom_routines/README.md]: Does a tutorial for this already exist? Can we link off to it (if on ai-platform-samples)?

Comment (nnegrey) [reply prediction/sklearn/structured/custom_routines/README.md]: Never mind, saw the pre-req

File: prediction/sklearn/structured/custom_routines/prediction/predict.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# Copyright 2019 Google LLC. All Rights Reserved.
```
Comment (nnegrey) [inline prediction/sklearn/structured/custom_routines/prediction/predict.py]: `# Copyright 2019 Google LLC`
No periods or All Rights Reserved

File: prediction/sklearn/structured/custom_routines/scripts/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+gcloud ai-platform models create ${MODEL_NAME} --regions=${REGION}
+
+echo "Second, creating the model version..."
+gcloud beta ai-platform versions create ${MODEL_VERSION} \
+  --model ${MODEL_NAME} \
+  --origin ${MODEL_DIR}/model \
+  --framework ${FRAMEWORK} \
+  --runtime-version=${RUNTIME_VERSION} \
+  --python-version=${PYTHON_VERSION} \
+  --package-uris ${CUSTOM_ROUTINE_PATH}
```
Comment (nnegrey) [inline prediction/sklearn/structured/custom_routines/scripts/deploy.sh]: Were all the used environment variables set in the `training` sample?

Comment (happyhuman) [reply prediction/sklearn/structured/custom_routines/scripts/deploy.sh]: Either in the training sample, or the steps before it (setup, or download).

File: prediction/sklearn/structured/custom_routines/prediction/predict.py (hunk 2 above)
Comment (happyhuman) [reply prediction/sklearn/structured/custom_routines/prediction/predict.py]: The period appears in all the files. Are you suggesting to remove all the periods from all the files?

Comment (nnegrey) [reply prediction/sklearn/structured/custom_routines/prediction/predict.py]: Yea.

Comment (happyhuman) [reply prediction/sklearn/structured/custom_routines/prediction/predict.py]: Done.

File: notebooks/deployment-manager/create_notebook.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google Inc. All rights reserved.
+# Copyright 2019 Google Inc. All Rights Reserved
```
Comment (nnegrey) [inline notebooks/deployment-manager/create_notebook.yaml]: Now it's just a game.

Comment (happyhuman) [reply notebooks/deployment-manager/create_notebook.yaml]: Hehehe, yeah, this is a never ending challenge.   :)
I think it is really done this time.

**Pull request merged**